rpc-jobs/RPC-Artifact-Build.yml

```diff
@@ -81,6 +81,16 @@
               Cleanup
     triggers:
       - timed: "{CRON}"
+      - github-pull-request:
+          org-list:
+            - rcbops
+          github-hooks: true
+          trigger-phrase: '.*recheck_cit_all.*|.*recheck_cit_{image}_{context}.*'
+          only-trigger-phrase: false
+          white-list-target-branches:
+            - "{branches}"
+          auth-id: "github_account_rpc_jenkins_svc"
+          status-context: 'CIT/{image}-{context}'
```

Comment:
Should probably add "artifact" to this so that it's different from the AIO test statuses.